geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -336,6 +336,12 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
     });
   }, this), 0);
 
+  if (this.appMymaps_.isEditable()) {
+    feature.set('__source__', 'mymaps');
+  } else {
+    feature.set('__source__', 'url');
+  }
+
```

Comment:
Did you see @pgiraud's comment? Why not just set the map id/uuid on the feature and use that as a way to know if the feature is a mymaps feature or not?